src/main/java/de/uhd/ifi/se/decision/management/jira/consistency/checktriggers/StatusClosedTrigger.java

```diff
@@ -4,10 +4,9 @@
 import com.atlassian.jira.event.type.EventType;
 
 
-public class StatusClosedTrigger implements ConsistencyCheckEventTrigger, TriggerChain<ConsistencyCheckEventTrigger> {
+public class StatusClosedTrigger extends TriggerChain {
 
 	private IssueEvent issueEvent;
-	private TriggerChain nextChainLink;
 
 	public StatusClosedTrigger(IssueEvent issueEvent) {
```

Comment:
Please rename this class to "StatusDoneTrigger" since the status is "done" not "closed".